### DISCUSSION TITLE
How to test a client side plugin?

### DISCUSSION THREAD

**Discussion by Vinccool96** [nuxt/nuxt#18767] [Questions] [answered]
I wrote a module, that creates a client side plugin. How do I test it in vite to see if it was indeed implemented?

**Comment by danielroe** [comment, marked as answer]
You can use the built in `createPage` function in test utils to run a browser e2e test, which would be the most accurate.

Or you could also try out the WIP module from Anthony and me: https://github.com/danielroe/nuxt-vitest
